README.md

```diff
@@ -51,23 +51,40 @@ pip install volttron
 volttron -vv -l volttron.log &
 ```
 
-Install and start the volttron-openadr-ven.
+3. Install and start the Volttron OpenADRVen Agent.
 
 ```shell
-vctl install volttron-openadr-ven --agent-config <path to agent config> --start
+vctl install volttron-openadr-ven --agent-config <path to agent config> \
+--vip-identity openadr.ven \
+--start
 ```
 
-View the status of the installed agent
+4. View the status of the installed agent
 
 ```shell
 vctl status
 ```
 
+5. Observe Data
+
+The OpenADR publishes events on the message bus. To see these events in the Volttron log file, install a [Listener Agent](https://pypi.org/project/volttron-listener/):
+
+```
+vctl install volttron-listener --start
+```
+
+Once installed, you should see the data being published by viewing the Volttron logs file that was created in step 2.
+To watch the logs, open a separate terminal and run the following command:
+
+```
+tail -f <path to folder containing volttron.log>/volttron.log
+```
+
 # Agent Configuration
 
 The required parameters for this agent are "ven_name" and "vtn_url". Below is an example of a correct configuration with optional parameters added.
 
-```jsonpath
+```yaml
```

Comment:
If you are going to use yaml...then you should change the content below to yaml as well.

Reply:
I'll change it to json to keep this documentation consistent with [readthedocs](https://volttron.readthedocs.io/en/develop/platform-features/config-store/configuration-store.html?highlight=json%20configuration#json). Also, agents use a JSON parser that can parse out the Python-style comments.